rest-api-spec/src/main/resources/rest-api-spec/api/cat.aliases.json

```diff
@@ -7,8 +7,8 @@
       "paths": ["/_cat/aliases", "/_cat/aliases/{name}"],
       "parts": {
         "name": {
-          "type" : "list",
-          "description" : "A comma-separated list of alias names to return"
+          "type" : "string",
+          "description" : "A string representing a list of alias name separated by commas"
```

Comment:
I think "comma-separated list" is more clear, as it is the common terminology for strings of this purpose, while "name separated by commas" seems unnecessarily verbose.